tests/datasource/test_datasource_serialization.py

```diff
@@ -64,13 +64,35 @@
         ),
     ],
 )
-def test_datasource_config_is_serialized(
-    datasource_config: DatasourceConfig, expected_serialized_datasource_config: dict
-):
-    """Datasource Config should be serialized appropriately with/without optional params."""
-    observed_dump = datasourceConfigSchema.dump(datasource_config)
-    assert observed_dump == expected_serialized_datasource_config
+class TestDatasourceConfigSerialization:
```

Comment:
Could we also update the tests for checkpoints and RBP?

Reply:
Yes, admittedly I only added some very minimal tests for this since it isn't being used yet and we hope to excise it based on my follow-up work.
I will refactor these tests so that we can properly parameterize it with various configs.